### ISSUE TITLE
Error when trying to deploy duplication rule

### ISSUE THREAD

**Issue by yurybond** [ForceCLI/force#506]
When run `force push` for duplicated rules as following:
`force push -f src/duplicateRules/Contact.XXXX.duplicateRule`
getting the following error:
```
Unknown type name 'duplicateRules' specified in package.xml
Test Failures - 0
ERROR: Some components failed deployment
```

**Comment by cwarden** [comment]
I've opened #508 to fix this.

**Comment by cwarden** [comment]
Fixed in #508
